src/platforms/common/performance/troubleshooting-performance.mdx

```diff
@@ -9,7 +9,7 @@ supported:
 description: "Learn how to troubleshoot your performance monitoring setup."
 ---
 
-If you need help managing transactions, you can read more here. If you need additional help, please view our forums. Customers on a paid plan may also contact support.
+If you need help managing transactions, you can read more here. If you need additional help, you can [ask on GitHub](https://github.com/getsentry/sentry/issues/new/choose), and customers on a paid plan may also contact support.
```

Comment:
This content shows up for all SDKs. This is probably the easiest way to get rid of the forum reference, but it does mean issues related to SDKs would end up on getsentry/sentry. 

We have a link to the repo based on the selected platform, so should be possible to render a link here that goes on the correct SDK repository